File: data/deepseek_coder_base_6.7B/readable_format_HPEC/fortran/file261.f90
subroutine print_array(n, x)
implicit none

DATA_TYPE, dimension(n, n) :: x
integer :: n
integer :: i, j

do i = 1, n
do j = 1, n
write(0, DATA_PRINTF_MODIFIER) x(j, i)
if (mod(((i - 1) * n) + j - 1, 20) == 0) then
write(0, *)
end if
end do
end do
write(0, *)
end subroutine

File: data/gpt-4o/readable_format_HPEC/cpp/file261.cpp
#include <iostream>
#define DATA_TYPE int
#define DATA_PRINTF_MODIFIER "%d"

void print_array(int n, DATA_TYPE x[][n]) 
{
 for (int i = 0; i < n; i++) 
 {
  for (int j = 0; j < n; j++) 
  {
   printf(DATA_PRINTF_MODIFIER, x[j][i]);
   if (((i * n) + j) % 20 == 0) 
   {
    printf("\n");
   }
  }
 }
 printf("\n");
}
